### PR TITLE
Convert AR to Yarn

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?

Use Yarn rather than NPM for Apps-Rendering.

The AR `package-lock.json` was imported using `yarn import`, so the versions of dependencies in use by all projects should be unchanged.

### Time to install deps
<table>
<tr>
	<th>AR package manager
	<th>First root-level `yarn`
	<th>Subsequent root-level `yarn`
<tr>
	<td>npm
	<td>77s
	<td>7s
<tr>
	<td>yarn
	<td>63s
	<td>0.4s
</table>

## Why?

As part of this [issue](https://github.com/guardian/dotcom-rendering/issues/6338
) we're taking steps to add AR to the Yarn workspace

### CODE REVIEW

**Review by OllysCoding — APPROVED** [review]
+1 if all works then hurrah!

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Ever thought about joining us?
 
 This repository contains the rendering logic for articles on theguardian.com. It is a monorepo with 3 projects, `apps-rendering`, `common-rendering` and `dotcom-rendering`.
 
-You should always `cd` into the correct subdirectory before running commands (e.g `make dev` for dotcom-rendering, or `npm run watch` for apps-rendering) except for storybook. Linting, imports, builds and github actions should work as before.
+You should always `cd` into the correct subdirectory before running commands (e.g `make dev` for dotcom-rendering, or `yarn watch` for apps-rendering) except for storybook. Linting, imports, builds and github actions should work as before.
```
Comment (SiAdcock) [inline README.md:10]: We should probably update the [README.md](https://github.com/guardian/dotcom-rendering/blob/main/apps-rendering/README.md) for apps-rendering also

Comment (sndrs) [reply README.md:10]: ooh thanks, not sure how we missed these, nice one

Comment (joecowton1) [reply README.md:10]: Thanks @SiAdcock ! I'd forgotten so push those changes 🤦

**Comment by joecowton1** [comment]
@guardian/apps-rendering if you're happy with this are we ok to merge?

**Review by georgeblahblah — APPROVED** [review]
Looks good to me! Thanks so much for taking this on

**Pull request merged**